molli/chem/structure.py

```diff
@@ -189,9 +195,6 @@ def dump_mol2(self, _stream: StringIO = None) -> None:
             dendrobine
             ...
         """
-        if _stream is None:
```

Comment:
Both here and line 222, it's not clear why this isn't in a similar format as done in `molecule.py` where there is a `with StringIO as stream`.

Reply:
Mistakenly though this was in `dump_mol2` not `dumps_mol2` all good